assets/src/components/app-menu.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import * as PropTypes from "prop-types";
+import showFormFields from "../util/dom-manipulation-utils";
+import "./app-menu.scss";
+
+/**
+ * App menu component.
+ *
+ * @class
+ * @param {object} props Props.
+ * @param {object} props.config Config for the app.
+ * @param {string} props.displayState The current display state.
+ * @param {Function} props.setDisplayState Set display state function.
+ * @returns {JSX.Element} App menu component.
+ */
+function AppMenu({ config, displayState, setDisplayState }) {
+  const onMenuClick = (e, display) => {
+    e.preventDefault();
+
+    e.stopPropagation();
+
+    switch (display) {
+      case "user_panel":
+        setDisplayState("userPanel");
+
+        showFormFields(false);
+
+        break;
+      default:
+        setDisplayState("maximized");
+
+        showFormFields(true);
+    }
+  };
+
+  return (
+    <div className="container-fluid app-menu">
+      <ul>
+        <li className="user-name">{config.user_name}</li>
```

Comment:
Instead of putting this data into config, add it to the user information endpoint in the drupal code and call this endpoint in this component.

https://github.com/itk-dev/drupal_webform_booking_module/blob/develop/itkdev_booking.routing.yml#L57

https://github.com/itk-dev/drupal_webform_booking_module/blob/develop/src/Controller/BookingController.php#L185